packages/dev/core/src/Materials/Textures/renderTargetTexture.ts

```diff
@@ -492,6 +492,9 @@ export class RenderTargetTexture extends Texture implements IRenderTargetTexture
         return this._renderTarget?._depthStencilTexture ?? null;
     }
 
+    /** @internal */
+    public _disableEngineStages = false;

```

Comment:
Wondering why we need this one ? (pure curiosity :-) )

Reply:
In the "frame graph path" of the code, we don't rely anymore on code injected by components at some specific location of the pipeline. As the shadow generator has not yet been reworked to be fully integrated inside the frame graph framework, it still uses `RenderTargetTexture` (whereas it should use `ObjectRenderer`), which executes a number of engine stages. That's why I had to create this property.

Reply:
Could we flag as TODO with the id or the issue ? to be sure it does not for through as it means the property will be removed soon ?